### ISSUE TITLE
machinectl: import-raw and import-tar not show progress / not respond

### ISSUE THREAD

**Issue by nosada** [systemd/systemd#9527]
**systemd version the issue has been seen with**
```
# systemctl --version
systemd 239
+PAM -AUDIT -SELINUX -IMA -APPARMOR +SMACK -SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 +SECCOMP +BLKID +ELFUTILS +KMOD -IDN2 +IDN +PCRE2 default-hierarchy=hybrid
```

```
$ pacman -Qs systemd
local/libsystemd 239.0-2
    systemd client libraries
local/systemd 239.0-2 (base-devel)
    system and service manager
```

**Used distribution**
Arch LInux

**Expected behaviour you didn't see**
- `machinectl import-raw` shows progress for importing image
- `machinectl import-raw` exits correctly after finished / failed to import image
- ditto with `machinectl import-tar`

**Unexpected behaviour you saw**
- `machinectl import-raw` doesn't show progress for importing image
- `machinectl import-raw` doesn't respond anymore after calling (see below)
- ditto with `machinectl import-tar`

```
# machinectl import-raw RAW_IMAGE
Enqueued transfer job 1. Press C-c to continue download in background.
(freezes)
```

Note that:
- it's possible to check progress of importing image in `machinectl list-transfers` or `systemctl status systemd-importd`
- At least in systemd 238.133 (238.133-4 in Arch LInux) above not occured

**Steps to reproduce the problem**
1. Create RAW image (for `import-raw`) or TAR archive (for `import-tar`)
    - I used https://github.com/systemd/mkosi
2. Do `machinectl import-raw` or `machinectl import-tar` for image created in `1.`
3. That's all

**Comment by yuwata** [comment]
Fix is waiting in #9536. Thanks.

**Comment by nosada** [comment]
@yuwata Thank you for your quick response and fixing!
